### PR TITLE
Update crun to 1.10

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Bump crun to the latest release.
#### Which issue(s) this PR fixes:

None
#### Does this PR have test?
None

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
@kubernetes-sigs/security-profiles-operator-maintainers PTAL

File: Dockerfile.ubi [hunk 1]
```diff
@@ -18,7 +18,7 @@
 # for more details
 
 # hash below referred to latest
-FROM registry.access.redhat.com/ubi8/go-toolset@sha256:61ffd49ec5868835ef1799bb7179765c48b99d909bfcf02d8c89d54074586356 as build
+FROM registry.access.redhat.com/ubi8/go-toolset@sha256:6c2c40207abfef2e46426f39c7fe0aca177b9da73d2b223bc0ddc7c5c56485c4 as build
```
Comment (saschagrunert) [inline Dockerfile.ubi:21]: Fixes CVE's

File: Dockerfile.ubi [hunk 2]
```diff
@@ -37,7 +37,7 @@ ARG STATIC_LINK=no
 RUN make
 
 # hash below referred to latest
-FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:8e68a739db66e514ce9ccfac7421e83d03180045e0a16ccf949cdac884150e5f
+FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:b93deceb59a58588d5b16429fc47f98920f84740a1f2ed6454e33275f0701b59
```
Comment (saschagrunert) [inline Dockerfile.ubi:40]: Fixes CVE's

**Comment by saschagrunert** [comment]
@Vincent056 PTAL, this fixes the CI

**Review by Vincent056 — APPROVED** [review]
/lgtm
Thanks for the PR!

**Pull request merged**